plugins/tor/tor_

```diff
@@ -0,0 +1,491 @@
+#!/usr/bin/python2
+
```

Comment:
Please add a documentation header here. This will help future readers and allows a proper visualization in the munin plugin gallery.

Other plugins should provide usable examples (e.g. [munin/munin_events](https://github.com/munin-monitoring/contrib/blob/master/plugins/munin/munin_events)).